docs/en/week07/07-3.md

```diff
@@ -0,0 +1,133 @@
+---
+lang-ref: ch.07-3
+title: Introduction to autoencoders
+lecturer: Alfredo Canziani
+authors: Vidit Bhargava, Monika Dagar
+date: 18 March 2021
+---
+# DALL-E: Creating Images from Text
```

Comment:
Why are there two titles?

![image](https://user-images.githubusercontent.com/2119355/119241096-0c9f9800-bb22-11eb-8869-118ec0214c0d.png)

Reply:
Perhaps use `##` for the subtitle?